### ISSUE TITLE
patch 1.29

### ISSUE THREAD

**Issue by marciii** [Josko/aura-bot#63]
Hey,

just found the changes that have to be applied for 1.29:
http://wiki.eurobattle.net/index.php?title=GHost_changes_in_patch_1.29
Will you do that? Otherwise I could give a try.

Best regards

**Comment by marciii** [comment]
^Tried this out - but thats not enough i guess (see the link i posted).

**Comment by Hedede** [comment]
Added PR #64 for 24-player support (must be used together with PR #63).

**Comment by marciii** [comment]
Worked for me, thanks @Hedede . The game adds the full amount of possible observers. When I try to !start the game though, immediately after the countdown has finished it says:
[GAME: <gn>] deleting player [<playername>]: has left the game voluntarily
[GAME: <gn>] is over (no players left)
[GAME: <gn>] saving game data to database
the players get kicked and the game crashes - before it actually loads. Does anyone experience the same?

**Comment by Hedede** [comment]
It worked for me. Try [v1.29 branch from my fork](https://github.com/Hedede/aura-bot/tree/v1.29).

**Comment by whiplikeflagela** [comment]
Hello, just set everything all up and having the same issue as marciii, players all get kicked as the countdown ends

**Comment by nielsAD** [comment]
You can't host "old" maps with 24 player lobbies. When you host an old map manually with the Warcraft 3 client, you also get a 12 player lobby. If you want to update your map for 24 player lobbies, load it in the world editor and save it again immediately. If you want to patch Aura to support lobbies of both sizes, you can check the version of the world editor in `map.cpp` to make sure 24 slots are supported (note: this isn't how the original client checks it, but it should work fine for our purpose).

**Comment by whiplikeflagela** [comment]
worked like a charm sir, thank you for all the help today.

**Comment by marciii** [comment]
yeah man, worked for me as well! Thank you guys for the fast answers!

**Comment by eap53** [comment]
Getting this issue when first opening aura bot:

[BNET: USWest] connecting to server [uswest.battle.net] on port 6112
[BNET: USWest] resolved and cached server IP address 12.129.236.17
[BNET: USWest] connected
[BNET: USWest] attempting to auth as Warcraft III: The Frozen Throne
[BNET: USWest] cd keys accepted
[TCPSOCKET] closed by remote host
[BNET: USWest] disconnected from battle.net
